app/components/shared/notification_component_preview.rb

```diff
@@ -0,0 +1,16 @@
+class Shared::NotificationComponentPreview < ViewComponent::Preview
+  layout "preview"
```

Comment:
I can't easily find a mention of the 'layout' keyword in ViewComponent documentation. What does it do?

Reply:
it actually something i will define in preview component config rather than at this level but its the layout template in design system all that stuff will be in different PR